integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheFunctionalityTest.java

```diff
@@ -45,6 +55,7 @@ public void testPanacheSerialisation() {
 
     @DisabledOnNativeImage
     @Test
+    @ActivateRequestContext
```

Comment:
Is the request context no longer active by default for tests, then?

Reply:
ah, good catch. I need to fix this, the test scope setup is being run from outside the scope of the deployment so it is not actually activating the scope.